### ISSUE TITLE
Webpack warning with v4.4.1

### ISSUE THREAD

**Issue by gtwilliams03** [chadly/react-bs-notifier#37]
Sorry to open another issue, but after upgrading to the latest, I get this warning when building in webpack:

```
WARNING in ./node_modules/react-bs-notifier/es/alert/styles.js
7:37-53 "export 'transitionStyles' was not found in '../alert-transition'
```

**Comment by gtwilliams03** [comment]
I am not positive, but it might be as simple as changing this statement in `/src/alert/styles.js` from:

```
import { transitionStyles } from "../alert-transition";
```

to:

```
import transitionStyles from "../alert-transition";
```

**Comment by chadly** [comment]
Sorry about that. Fixed in #38 & `v4.4.2`

**Comment by gtwilliams03** [comment]
Thank you for your quick response!
